README.md

```diff
@@ -48,6 +48,7 @@ Do the following steps to setup TON Indexer:
   * `TON_INDEXER_API_PORT`: a port to expose. You need check if this port is busy. Use different ports in case of multiple indexer instances on the same host. Default: `8081`.
   * `TON_INDEXER_TON_HTTP_API_ENDPOINT`: an endpoint to [ton-http-api](https://github.com/toncenter/ton-http-api). Indexer API contains several proxy methods to ton-http-api service: `/message`, `/runGetMethod`, `/estimateFee`, `/account`, `/wallet`. If the variable is not set these methods will not be included. Default: `<blank>`
   * `TON_INDEXER_WORKERS`: number of API workers. Default: `4`.
+  * `TONCENTER_ENV`: mainnet or testnet. Default: `mainnet`.
```

Comment:
The environment variables TONCENTER_ENV, IMAGE_TAG together with a script "swarm-deploy.sh" are used to deploy in Docker Swarm mode. Sometimes the images of API and worker for mainnet and for testnet should be different.
The mistake in image in such case may cause critical issues. So, absence of default value in Compose files is the expected behavior. 

Instead of specifying the default value I suggest to add these environment variables in script "configure.sh" and specify default values there.